components/formats-bsd/src/loci/formats/in/TileJPEGReader.java

```diff
@@ -58,8 +58,7 @@
 
   // -- Fields --
 
-  private JPEGTileDecoder decoder;
-  private JPEGTurboService service;
+  private transient JPEGTurboService service;
```

Comment:
As a data point in deciding, I was just forced to break bfmemo compatibility in a _patch_ release. Using three boxes, the estimated time to recalculate all the memo files is roughly 100 hours.